src/connector/src/sink/log_store.rs

```diff
@@ -304,6 +326,27 @@ where
     }
 }
 
+pub struct PausableLogWriter<W> {
+    inner: W,
+    is_paused: watch::Sender<bool>,
+}
```

Comment:
a little curious about where the struct used

Reply:
Same here. I didn't see this struct being used?